components/script/dom/htmlinputelement.rs

```diff
@@ -875,6 +875,24 @@ impl HTMLInputElement {
                 content.strip_newlines();
                 content.strip_leading_and_trailing_ascii_whitespace();
             }
+            atom!("color") => {
+                let mut textinput = self.textinput.borrow_mut();
+                let content = textinput.get_content();
+                let is_valid = {
+                    let mut chars = content.chars();
+                    if content.len() == 7 && chars.next() == Some('#') {
+                        chars.fold(true, |v, c| v && c.is_digit(16))
```

Comment:
Instead of using `fold` here, we can use `all`.

Reply:
Nice, didn't know about that :)